README.md

```diff
@@ -71,6 +71,14 @@ bun run pgo.js
 
 And the binary will be available at `target/[target-triple]/pgo/oha`.
 
+## X-CMD
+
+You can install with [x-cmd](https://www.x-cmd.com).
+
+```sh
+x env use oha
+```
+
```

Comment:
Can you move it to the `Install` section?

Reply:
> Can you move it to the `Installation` section?

ok, I moved it before the Containerized section, please take another look.

Reply:
> Can you move it to the `Installation` section?你能把它移到`Installation`部分吗？

Or should I put it before `On Arch Linux`, alongside "cargo"?